docs/ai/conceptual/agents.md

```diff
@@ -0,0 +1,91 @@
+---
+title: Agents
+description: Introduction to agents
+author: luisquintanilla
+ms.author: luquinta
+ms.date: 10/01/2025
+ms.topic: concept-article
+---
+
+# Agents
+
+This article introduces the core concepts behind agents, why they matter, and how they fit into workflows, setting you up to get started building agents in .NET.
+
+## What are agents?
+
+**Agents are systems that accomplish objectives.**
+
+![Components of an agent](../media/agents/agent-components.png)
+
+Agents become more capable when equipped with the following:
+
+- **Reasoning and decision-making**: Powered by LLMs, search algorithms, or planning and decision-making systems.
+- **Tool usage**: Access to Model Context Protocol (MCP) servers, code execution, and external APIs.
+- **Context awareness**: Informed by chat history, threads, vector stores, enterprise data, or knowledge graphs.
+
+These capabilities allow agents to operate more autonomously, adaptively, and intelligently.
+
+## What are workflows?
+
+As objectives grow in complexity, they need to be broken down into manageable steps. That’s where workflows come in.
+
+**Workflows define the sequence of steps required to achieve an objective.**
+
+Imagine you're launching a new feature on your business website. If it's a simple update, you might go from idea to production in a few hours. But for more complex initiatives, the process might include:
+
+- Requirement gathering
+- Design and architecture
+- Implementation
+- Testing
+- Deployment
+
+A few important observations:
+
+- Each step may contain subtasks.
+- Different specialists may own different phases.
+- Progress isn’t always linear. Bugs found during testing may send you back to implementation.
+- Success depends on planning, orchestration, and communication across stakeholders.
+
+### Agents + Workflows = Agentic Workflows
+
+Workflows don’t require agents, but agents can supercharge them.
+
+When agents are equipped with reasoning, tools, and context, they can optimize workflows.
+
+This is the foundation of multi-agent systems, where agents collaborate within workflows to achieve complex goals.
+
+### Workflow orchestration
+
+Agentic workflows can be orchestrated in a variety of ways. The following are a few of the most common.
+
+#### Sequential
+
+Agents process tasks one after another, passing results forward.
+
+![Sequential agent orchestration: Task Input → Agent A → Agent B → Agent C → Final Output](../media/agents/sequential-workflow.png)

```

Comment:
Can you trim this image down?

<img width="1363" height="757" alt="image" src="https://github.com/user-attachments/assets/b2655cb3-38f6-460b-8802-2c10ef410f39" />